lib/main.js

```diff
@@ -51,13 +52,9 @@ function post(options) {
 }
 
 module.exports = class Client {
-    constructor(options = {}) {
-        this.options = Object.assign(
-            {
-                buildServerUri: 'http://127.0.0.1:7100',
-            },
-            options
-        );
+    constructor({ serverId, buildServerUri =  'http://127.0.0.1:7100' } = {}) {
```

Comment:
OP says "removed buildServerUri default value", but it's still here?

Can you throw if it's missing?

Reply:
pushed.